src/main/resources/templates/car/car-update.html

```diff
@@ -0,0 +1,178 @@
+
+                        <div class="form-group"
+                             th:classappend="${#fields.hasErrors('make')}? 'has-error':''">
+
+                            <label for="make" class="control-label">Make</label>
+
+                            <input id="make" class="form-control" type="text" th:field="*{make}" aria-describedby="nameHelp" required>
+                            <p class="error-message"
+                               th:each="error: ${#fields.errors('make')}"
+                               th:text="${error}">Validation error</p>
+                        </div>
+
+                        <div class="form-group"
+                             th:classappend="${#fields.hasErrors('model')}? 'has-error':''">
+
+                            <label for="model" class="control-label">Model</label>
+
+                            <input
+                                    id="model" class="form-control" th:field="*{model}"/>
+                            <p class="error-message"
+                               th:each="error: ${#fields.errors('model')}"
+                               th:text="${error}">Validation error</p>
+                        </div>
+
+                        <div class="form-group"
+                             th:classappend="${#fields.hasErrors('bodyType')}? 'has-error':''">
+
+                            <label for="bodyType" class="control-label">Body Type</label>
+
+                            <input
+                                    id="bodyType" class="form-control" th:field="*{bodyType}"/>
+                            <p class="error-message"
+                               th:each="error: ${#fields.errors('bodyType')}"
+                               th:text="${error}">Validation error</p>
+                        </div>
+
+                        <div class="form-group"
+                             th:classappend="${#fields.hasErrors('year')}? 'has-error':''">
+
+                            <label for="year" class="control-label">Year</label>
+
+                            <input
+                                    id="year" class="form-control" th:field="*{year}"/>
+                            <p class="error-message"
+                               th:each="error: ${#fields.errors('year')}"
+                               th:text="${error}">Validation error</p>
+                        </div>
+
+                        <div class="form-group"
+                             th:classappend="${#fields.hasErrors('color')}? 'has-error':''">
+
+                            <label for="color" class="control-label">Color</label>
+
+                            <input
+                                    id="color" class="form-control" th:field="*{color}"/>
+                            <p class="error-message"
+                               th:each="error: ${#fields.errors('color')}"
+                               th:text="${error}">Validation error</p>
```

Comment:
You don't need to put "Validation error" here. This is because `th:text="${error}"` will add the text for you.